device/iot-device-samples/device-reconnection-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/SampleParameters.java

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package samples.com.microsoft.azure.sdk.iot;
+
+import com.azure.core.http.policy.HttpLogDetailLevel;
```

Comment:
Is this import necessary?

Reply:
good catch, left behind from a different check.